dotCMS/src/main/java/com/dotcms/security/apps/AbstractProperty.java

```diff
@@ -24,6 +25,10 @@ public T getValue() {
     }
 
     public boolean isHidden() {
+        return UtilMethods.isSet(hidden) ? hidden : false;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Ternary operators that can be simplified with || or &&](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5358201)